scripts/generate_json_docs.py

```diff
@@ -296,9 +302,35 @@ def clean_source_path(module):
     return '%s.py' % (source_id.replace('.', '/'),)
 
 
+def get_examples_from_docstring(doc_str):
+    """Parse doctest style code examples from a docstring."""
+    examples = docstring_test_parser.get_examples(doc_str)
+    example_str = ''
+    for example in examples:
+        example_str += '%s' % (example.source,)
+        example_str += '%s' % (example.want,)
+
+    return example_str.replace('<', '&lt;').replace('>', '&gt;')
```

Comment:
Sorry this got folded up but I answered it before.
https://github.com/GoogleCloudPlatform/gcloud-python/pull/2039#discussion_r75166310